script/pipeline.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+anvil --fork-url $ETH_RPC_URL_MAINNET >/dev/null 2>&1 &
+ANVIL_PID=$!
+trap "kill $ANVIL_PID >/dev/null 2>&1 || true" EXIT
+# wait for anvil to be ready
+until curl -s http://localhost:8545 >/dev/null 2>&1; do sleep 0.5; done
+
+DEPLOYER=0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba
+cast rpc anvil_impersonateAccount $DEPLOYER --rpc-url http://localhost:8545
+forge script script/DeployHypMinter.s.sol --rpc-url http://localhost:8545 --unlocked --sender $DEPLOYER --broadcast --sig "run()" -vvvv
```

Comment:
Do we need both `--unlocked` and -`-sender`? Maybe neither are needed because of the account impersonation?

From `forge script --help`

```
 --unlocked
          Send via `eth_sendTransaction` using the `--from` argument or `$ETH_FROM` as sender
          
 --sender <ADDRESS>
          The address which will be executing tests/scripts
```